examples/bvh_driver/bvh_driver.cpp

```diff
@@ -269,11 +301,14 @@ int main(int argc, char *argv[])
     ;
   // clang-format on
   bpo::variables_map vm;
-  bpo::parsed_options opts = bpo::command_line_parser(argc, argv)
-                                 .options(desc)
-                                 .allow_unregistered()
-                                 .run();
-  bpo::store(opts, vm);
+  bpo::parsed_options parsed = bpo::command_line_parser(argc, argv)
+                                   .options(desc)
+                                   .allow_unregistered()
+                                   .run();
+  bpo::store(parsed, vm);
+  CmdLineArgs pass_further{
+      bpo::collect_unrecognized(parsed.options, bpo::include_positional),
+      argv[0]};
```

Comment:
Wait so we are using boost::PO for some arguments and the new code for others? What's so special about the new arguments?

Reply:
The parsings goes like this:
1. Kokkos parse its own args and consume them like it should
2. We then parse the remaining arguments but cannot throw when args are not recognized because they might be some that go to Google benchmark
3. Benchmark consume their own args
4. We ensure all arguments have been consumed using BPO

Reply:
You might ask why aren't we doing 1, 3, (2+4)

It is because of Google benchmark calling `std::exit(0)` when `--help` is passed :(

For reference: https://github.com/google/benchmark/blob/33d440465041fe83b831f537b9646756655806f0/src/benchmark.cc#L426